# GM-driven metabolic alterations in AD mice
The gut microbiota (GM) and its metabolites affect the host nervous system and are involved in the pathogeneses of various neurological diseases. However, the specific GM alterations under pathogenetic pressure and their contributions to the “microbiota–metabolite–brain axis” in Alzheimer’s disease (AD) remain unclear. Here, we investigated the GM and the fecal, serum, cortical metabolomes in APP/PS1 and wild-type (WT) mice, revealing distinct hub bacteria in AD mice within scale-free GM networks shared by both groups. We present the major bioinformatic scripts used in the study. <br>
* GM network construction
* Functional prediction with 16S sequences
* Correlations between GM and metabolites
* Mediation effect analysis
# Bioinformatic contents
## GM network construction
Applying the sparCC software, we constructed the GM co-occurrence networks. The related files and scripts can be found under the  ```Network``` folder.<br>


## Functional prediction with 16S sequences
Applying the PICRUST2 software, we predicted the functions for the GM with the taxonomical annotated 16S sequences. The related files and scripts can be found under the  ```Function``` folder.<br>

## Correlations between GM and metabolites
Applying the mmVEC software, we deduced the relations between GM and metabolites. The related files and scripts can be found under the  ```GM-MS correlation``` folder.<br>

## Mediation effect analysis
We adopted mediation analysis to decipher the intricate connections between GM alterations and disease occurrence via the mediation of metabolites. The related files and scripts can be found under the  ```Mediation``` folder.<br>

# Contact

Feel free to open an issue in Github or contact xjy005351@siat.ac.cn if you have any problem in practicing the scripts.

# License

This project is licensed under the MIT License - see the LICENSE file for details.
